src/Input.test.jsx

```diff
@@ -0,0 +1,41 @@
+import { render, fireEvent } from '@testing-library/react';
+
+import Input from './Input';
+
+describe('Input', () => {
+  const handleClick = jest.fn();
+
+  const handleChange = jest.fn();
+
+  const renderInput = () => (
+    render(
+      <Input
+        value="아무거나 하기"
+        onClick={handleClick}
+        onChange={handleChange}
+      />,
+    )
```

Comment:
```suggestion
    render((
      <Input
        value="아무거나 하기"
        onClick={handleClick}
        onChange={handleChange}
      />
    ));
```

괄호를 2개로 감싸주시면 콤마를 없앨 수 있어요.